openapi-codegen/core/src/test/scala/sttp/tapir/codegen/TestHelpers.scala

```diff
@@ -114,6 +114,15 @@ object TestHelpers {
       |                type: string
       |      tags:
       |        - Bookshop
+      |    delete:
```

Comment:
That this will now work..

![Feels good](https://www.pngkey.com/png/detail/154-1546056_213kib-1600x1564-feels-good-feels-good-meme-png.png)